### PR TITLE
Updates to Amazon Sales Channel Home

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updated the Amazon Sales Channel Home (Store View, Learning and Preparation, and Attributes) section of the 4.0+ toc.

## Affected documentation pages

- TOC: Amazon Sales Channel Home
- additional pages affected by Home section

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

staging build #386 in progress at submission

### CODE REVIEW

File: src/sales-channels/asc/managing-attributes.md [hunk 1]
```diff
@@ -5,31 +5,17 @@ title: Managing Attributes
...
 
-Once attribute matching is set up, Magento can communicate product information back and forth with Amazon. If you have Amazon product listings, Magento can import your Amazon products and details into your Magento catalog, allowing you to manage your Amazon listings from a single, central catalog of products.
+Once attribute mapping is set up, Magento can communicate product information back and forth with Amazon. If you have Amazon product listings, Magento can import your Amazon products and details into your Magento catalog, allowing you to manage your Amazon listings from a single, central catalog of products.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-attributes.md]:
```suggestion
When attribute mapping is set up, Magento can communicate product information back and forth with Amazon. If you have Amazon product listings, Magento can import your Amazon products and details into your Magento catalog, allowing you to manage your Amazon listings from a single, central catalog of products.
```

File: src/sales-channels/asc/managing-attributes.md [hunk 2]
```diff
@@ -5,31 +5,17 @@ title: Managing Attributes
...
 
-The Amazon Sales Channel allows you to access, review, create, and manage attributes, as needed. If you add a new attribute to your Magento catalog, you may need to update those values across all products. For more information on Magento attribute sets and values, see [Product Attributes]({% link catalog/product-attributes.md %}).
+Amazon Sales Channel allows you to access, review, create, and manage attributes, as needed, in the [_Attributes_ view]({% link sales-channels/asc/attributes-view.md %}) on the Amazon Sales Channel home page. If you add a new attribute to your Magento catalog, you may need to update those values across all products.
 
-### Attribute Options
+For more information on Magento attribute sets and values, see
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-attributes.md]:
```suggestion
For more information on Magento attribute sets and values, see:
```

File: src/sales-channels/asc/managing-stores.md [hunk 3]
```diff
@@ -1,41 +1,36 @@
...
 
-When accessing your Amazon Sales Channel, the _Amazon Stores_ tab opens by default on Amazon Sales Channel Home. In the _Admin_ sidebar, go to **Marketing** >  _Channels_ > **Amazon**.
+When viewing the Amazon Sales Channel home page, the _Amazon Stores_ view opens by default.
 
-The _Amazon Stores_ tab provides a list of your Amazon stores along with some basic statistics and management options. Displayed statistics include each store's status, date created, last updated, available management actions, and defined region. Management options (available under **Select** in the _Actions_ column) include:
+![]({% link sales-channels/asc/assets/amazon-sales-channel-home-tabs.png %}){: .zoom}
+_Amazon Stores view_
 
-- **Manage Store** - The action for this option is dependent on the status of the store selected.
+The _Amazon Stores_ view shows a "store card" for each of your Amazon stores along with some basic statistics and management options. Summary information shown in each card include each store's status, date created, last updated date, listings that need attention (example: Incomplete Listings) and the assigned Magento website.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-stores.md]:
```suggestion
The _Amazon Stores_ view shows a "store card" for each of your Amazon stores along with some basic statistics and management options. Summary information shown in each card includes each store's status, date created, last updated date, listings that need attention (example: Incomplete Listings) and the assigned Magento website. 
```

File: src/sales-channels/asc/managing-stores.md [hunk 4]
```diff
@@ -1,41 +1,36 @@
...
 
-    Changing an Inactive store to Active status will activate listings and order activity for the store, using the stores current setup (listing settings, price rules, overrides, etc.)
-    
-    Changing a store's status from `Active` to `Inactive` will suspend listings and order activity for the store. An Inactive store will retain all settings and listings, but will temporarily stop the synchronization of pricing, quantity, and order management until the store is changed to `Active` status. This feature allows you to control your store activity at the Regional level without the need to recreate or reintegrate your Amazon store or the loss of historical order and sales data.
+- Click **Action** to change a store's status or delete a store.
 
-- **Delete Store** - Choose when you wish to delete an existing Amazon store and its integration settings with your Amazon Seller Central account. Deleting the account will remove the store from Amazon Sales Channel, along with all account settings, listings, logs and other information related to this store. The account cannot be retrieved after deletion.
+   - **Activate** / **Deactivate** - Allows you to change a store's status.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-stores.md]: This isn't parallel with the  preceding two bullet items.

File: src/sales-channels/asc/managing-stores.md [hunk 5]
```diff
@@ -1,41 +1,36 @@
...
 
-{:.bs-callout-info}
-To change the website assigned to the store during integration, you must delete the store and add the store again with the different website through the onboarding process.
+      Changing an `Inactive` store to `Active` status will activate listings and order activity for the store, using the stores current store settings (listing settings, price rules, overrides, etc.)
+
+      Changing a store's status from `Active` to `Inactive` status will suspend listings and order activity for the store. An Inactive store will retain all store settings and listings, but will temporarily stop the synchronization of pricing, quantity, and order management until the store is changed back to `Active` status. This feature allows you to control your store activity at the Regional level without the need to recreate or reintegrate your Amazon store or the loss of historical order and sales data.
 
-You will notice some [workspace controls]({% link sales-channels/asc/workspace-controls.md %}) that are common throughout the Amazon Sales Channel.
+   - **Delete** - Allows you to delete a store that is no longer needed.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-stores.md]: Also not parallel

File: src/sales-channels/asc/managing-stores.md [hunk 6]
```diff
@@ -1,41 +1,36 @@
...
+|Top section|Includes: <br/>The region icon for the store, defined during store integration.<br/> The assigned **Magento Website**, defined during store integration.<br/>The **Status** of your store. Options: **Active** - Store integration is complete and verified with Amazon and is available for sales activity. **Inactive** - Store integration is complete, but is not in use or available for sales activity. When `Inactive`, your Amazon sales are paused. When `Active`, sales revenue and additional settings save to update prior to activating.<br/>The **Last Updated** date of the most recent change to the Amazon store's setup.<br/>The **Created** date when the Amazon store was created in Amazon Sales Channel.|
+|Middle Section|Includes a store activity summary chart for the last 30 days and includes and alert for any listings that need attention.|
+|Bottom section|Includes the View Store and Action options.<br/>Click **View Store** to open the store's [dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).<br/> Click **Actions** to Activate, Deactivate, or Delete a store.|
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-stores.md]:
```suggestion
|Bottom section|Includes the View Store and Action options.<br/>Click **View Store** to open the store's [dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).<br/> Click **Actions** to activate, deactivate, or delete a store.|
```

File: src/sales-channels/asc/order-settings.md [hunk 7]
```diff
@@ -10,16 +10,16 @@ Order Settings define how Amazon orders are imported into and processed in Magen
...
+When an order is created on Amazon, it is not immediately imported into Amazon Sales Channel. Amazon assigns a `Pending` status to newly created orders. Once Amazon verifies the order and payment method, Amazon changes the order's status to `Unshipped`. This status change triggers Amazon Sales Channel to import the order and create a corresponding order. Amazon Sales Channel creates a matching order that corresponds to the Amazon order. Orders created in Amazon Sales Channel display and can be managed in the [_Orders_]({% link sales-channels/asc/managing-orders.md %}) tab on the Amazon Sales Channel home page.
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
When an order is created on Amazon, it is not immediately imported into Amazon Sales Channel. Amazon assigns a `Pending` status to newly created orders. After it verifies the order and payment method, Amazon changes the order's status to `Unshipped`. This status change triggers Amazon Sales Channel to import the order and create a corresponding order. Amazon Sales Channel creates a matching order that corresponds to the Amazon order. Orders created in Amazon Sales Channel are displayed and can be managed in the [_Orders_]({% link sales-channels/asc/managing-orders.md %}) tab on the Amazon Sales Channel home page.
```

File: src/sales-channels/asc/order-settings.md [hunk 8]
```diff
@@ -10,16 +10,16 @@ Order Settings define how Amazon orders are imported into and processed in Magen
...
 
 {:.bs-callout-info}
 When an order is received from Amazon, the extension is set to Create Magento Order, and the ordered item SKU cannot be matched to an existing Magento catalog product, the order synchronization process: <br/>- creates a Magento catalog product with the SKU<br/>- Sets the product status to `Disabled` and the inventory setting to `Do Not Manage Inventory`<br/>- Creates the Magento order
 
-# To configure Order Settings:
+# onfigure Order Settings
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
# Configure Order Settings
```

File: src/sales-channels/asc/order-settings.md [hunk 9]
```diff
@@ -43,13 +43,13 @@ When an order is received from Amazon, the extension is set to Create Magento Or
     - **Build Using Amazon Order Number** - Choose when you want to create the Magento order number using the corresponding Amazon-assigned order number.
 
     {:.bs-callout .bs-callout-info}
-    Once an order is imported, the Amazon Order Number and the Magento Order Number display in the [Orders tab]({% link sales-channels/asc/managing-orders.md %}).
+    Once an order is imported, the Amazon Order Number and the Magento Order Number display in the [_Orders_ tab]({% link sales-channels/asc/managing-orders.md %}).
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
    After an order is imported, the Amazon Order Number and the Magento Order Number are displayed in the [_Orders_ tab]({% link sales-channels/asc/managing-orders.md %}).
```

**Comment by hickskenh** [comment]
running tests

**Pull request merged**